wiki/Beatmaps/en.md

```diff
@@ -1,207 +1,86 @@
 # Beatmaps
 
-## What are "Beatmaps"?
+Beatmaps, or _map_ for short, are game levels in [osu!](/wiki/osu!_glossary), like the stages in the [DS games](/wiki/Glossary/#ds-games).
+In their most basic form, beatmaps have a song various [hit objects](/wiki/hit_objects) placed according to the rhythm of the music.
+They can also have a background image, video, and/or [storyboard](/wiki/storyboards).
 
-*Beatmaps listing page: <https://osu.ppy.sh/p/beatmaplist>*
+Creators of beatmaps are called beatmappers or mappers for short.
+Beatmaps are uploaded for everyone to see and play with using the [BSS](/wiki/BSS), and are then [modded](/wiki/modded).
+If and once a beatmap is deemed good enough, it is [ranked](#ranked) or, if fulfilling certain criteria, [approved](#approved).
```

Comment:
`Once a beatmap is deemed good enough, it is [ranked](#ranked) or if fulfilling certain criteria, [approved](#approved).`